### DISCUSSION TITLE
Marker screen position (x,y)

### DISCUSSION THREAD

**Discussion by Aboidrees** [fleaflet/flutter_map#1483] [Q&A] [answered]
Hi,
How can I get the marker point (x, y) on the screen?  Not the GeoLocation(LAT, LNG).

**Comment by JaffaKetchup** [comment, marked as answer]
Hi @Aboidrees,
The `onTap` callback takes two arguments. The first is a `TapPosition` which contains information about the position tapped on screen (x, y).
Also, [`latLngToScreenPoint`](https://pub.dev/documentation/flutter_map/latest/flutter_map/MapController/latLngToScreenPoint.html) is a method accessible from the `MapController`, which does the same thing.
